nixos/platform/default.nix

```diff
@@ -322,6 +322,23 @@ in {
       # overrides it
       cron.enable = fclib.mkPlatform true;
 
+      fail2ban.enable = fclib.mkPlatform true;
+      fail2ban.ignoreIP =
+        [
+          # loopback
+          "127.0.0.1/8"
+          "::1"
+
+          # rfc1918 addresses
+          "10.0.0.0/8"
+          "172.16.0.0/12"
+          "192.168.0.0/16"
+        ] ++
+        config.flyingcircus.static.firewall.trusted ++
+        (lib.flatten
+          (builtins.map (v: builtins.attrNames v.networks)
+            (builtins.attrValues config.flyingcircus.enc.parameters.interfaces)));
```

Comment:
Tests are breaking now which don't have the expected attributes: https://hydra.flyingcircus.io/jobset/platform-prs/pr-755#tabs-errors

We use `lib.attrByPath [ "parameters" ...` in platform code.

Reply:
See [98465b7](https://github.com/flyingcircusio/fc-nixos/pull/755/commits/98465b7303b36371b59d4f6e416f80d853ea0eee).